### PR TITLE
[Sample] [Graphics] [Compose] UltraHDR image rendering in Compose. 

### PR DESCRIPTION
**Pull request by MayuriKhinvasara**
**Description**
This CL contains sample code for rendering UltraHDR Images in Compose (on API 34 and above).

**How Has This Been Tested?**
Test Configuration https://github.com/android/platform-samples/pull/1 - Pixel 7 Pro API 34 Beta (Android 14 Beta)

**Checklist**
I have performed a self-review of my code
I have commented my code, particularly in hard-to-understand areas

**Screenshots**
image

### CODE REVIEW

**Review by madebymozart — APPROVED** [review]
LGTM

**Review by Emilycroxall1996 — CHANGES_REQUESTED** [review]
Na

File: samples/graphics/pdf/build.gradle.kts [hunk 1]
```diff
@@ -26,4 +26,7 @@ android {
 
 dependencies {
     // Add samples specific dependencies
+    implementation(libs.compose.bom)
```
Comment (yrezgui) [inline samples/graphics/pdf/build.gradle.kts:29]: This sample is unrelated

File: samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.viewinterop.AndroidView
+import androidx.fragment.app.Fragment
+import com.example.platform.graphics.ultrahdr.databinding.DisplayingUltrahdrBinding
+import com.google.android.catalog.framework.annotations.Sample
+
+@Sample(
+    name = "Displaying UltraHDR (Compose)",
+    description = "This sample demonstrates displaying an UltraHDR image in a Compose View .",
```
Comment (yrezgui) [inline samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt:50]: Typo at the end "."

File: samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    documentation = "https://developer.android.com/guide/topics/media/hdr-image-format",
+    tags = ["UltraHDR", "Compose"],
+)
+@RequiresApi(Build.VERSION_CODES.UPSIDE_DOWN_CAKE)
+class DisplayingUltraHDRCompose : Fragment() {
```
Comment (yrezgui) [inline samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt:55]: None of this is required for Compose sample, have a look at this sample: https://github.com/android/platform-samples/blob/ccb38522314afb30a14aba74135f1668f91dd4ee/samples/storage/src/main/java/com/example/platform/storage/photopicker/PhotoPicker.kt#L63-L69

File: samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    }
+
+}
+
+@Composable
+fun rendercomposeView() {
+
+    val ultra_hdr_image = "gainmaps/night_highrise.jpg"
+    val stream = LocalContext.current.assets.open(ultra_hdr_image)
+    val bitmap = BitmapFactory.decodeStream(stream)
```
Comment (yrezgui) [inline samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt:87]: Decode the image in a coroutine, not in the main thread

**Review by yrezgui — CHANGES_REQUESTED** [review]

File: samples/graphics/pdf/build.gradle.kts (hunk 1 above)
Comment (MayuriKhinvasara) [reply samples/graphics/pdf/build.gradle.kts]: Thanks. Removed this dependency.

File: samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt (hunk 2 above)
Comment (MayuriKhinvasara) [reply samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt]: Ack and removed

File: samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt (hunk 3 above)
Comment (MayuriKhinvasara) [reply samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt]: Ack and removed the fragment.

File: samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt (hunk 4 above)
Comment (MayuriKhinvasara) [reply samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRCompose.kt]: Added the asset loading and image decoding in a background thread.

File: samples/graphics/pdf/build.gradle.kts [hunk 5]
```diff
@@ -26,4 +26,6 @@ android {
 
 dependencies {
     // Add samples specific dependencies
+    implementation(libs.compose.runtime.runtime)
```
Comment (yrezgui) [inline samples/graphics/pdf/build.gradle.kts:29]: This is not needed

**Review by yrezgui — CHANGES_REQUESTED** [review]
Make sure you're updating only the right sample dependencies

File: samples/graphics/pdf/build.gradle.kts (hunk 5 above)
Comment (MayuriKhinvasara) [reply samples/graphics/pdf/build.gradle.kts]: Thanks. remived unnecessary dependencies

**Comment by MayuriKhinvasara** [comment]
> Make sure you're updating only the right sample dependencies

Updated, Ack

**Comment by madebymozart** [comment]
@yrezgui Is there any more blockers on this one?

**Review by yrezgui — APPROVED** [review]
